core/domain/user_services.py

```diff
@@ -2330,7 +2330,7 @@ def can_submit_question_suggestions(user_id: str) -> bool:
 
 
 def allow_user_to_review_translation_in_language(
-    user_id: str, language_code: str
+    user_id: str, language_code: Optional[str]
```

Comment:
I did this change because here https://github.com/oppia/oppia/blob/934e23416ebb513935c124ed2be93e26a2632785/core/controllers/contributor_dashboard_admin.py#L106
Language_code can be none. At first, it seems that language_code should not be none but the method just above `allow_user_to_review_translation_in_language` in contributor_dashboard is accepting optional language_code https://github.com/oppia/oppia/blob/934e23416ebb513935c124ed2be93e26a2632785/core/controllers/contributor_dashboard_admin.py#L101

So I'm not sure is Optional language_code ok or not.

Reply:
In both cases it can't be optional (conceptually). Could you please modify accordingly?

Reply:
Raised an exception PTAL!